src/swap/components/SwapAmountInput.tsx

```diff
@@ -121,6 +127,60 @@ export function SwapAmountInput({
     }
   }, [token, setToken]);
 
+  useEffect(() => {
```

Comment:
Just for readibility for the code let's add a line of comment to each `useEffect`, so when reading this later we don't forget what's are all about.

Reply:
this logic was removed